packages/services/api/src/modules/audit-logs/module.graphql.ts

```diff
@@ -0,0 +1,236 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  interface AuditLog {
```

Comment:
Also, add events for:
- (billing) Subscription created
- (billing) Subscription updated (limits changed) 
- (billing) Subscription canceled 
- (support) Create support ticket 
- (support) Update/reply support ticket
- (lab) CRUD collection
- (lab) CRUD query in collection
- (target) create/delete CDN token
- (target) create/delete registry token 
- (target) Conditional Breaking Changes + Base Schema (both under the same target settings) 
- (org settings) Integration connect/disconnect
- (org settings) OIDC settings changes (under org settings)  
- (org settings) make sure policy "Allow projects to override or disable rules" recorded